composer.json

```diff
@@ -36,7 +36,7 @@
     "oat-sa/extension-tao-community": "10.3.1",
     "oat-sa/extension-tao-funcacl": "7.2.2",
     "oat-sa/extension-tao-dac-simple": "7.8.0",
-    "oat-sa/extension-tao-itemqti": "29.22.2",
+    "oat-sa/extension-tao-itemqti": "29.25.1",
```

Comment:
Is this the proper next version? Should not it be `29.22.3` or `29.22.2.1` ?

Reply:
done! :) thanks gabriel!!